<?xml version="1.0"?>
<!-- vim: set fenc=utf-8 et sts=4 ft=xml : -->
<launch>

    <node
        machine="navigate"
        pkg="executive"
        type="master_executive.py"
        name="master_executive"
        output="log">
        <param name="precached_sample_distance" type="double" value="10.0"/>
        <param name="home_pursuit_complete_distance" type="double" value="4.0"/>
        <param name="maximum_pursuit_error" type="double" value="0.5"/>

        <!--
        <remap from="img_point" to="filtered_points"/>
        -->
    </node>

    <node
        machine="search"
        pkg="detection_filter"
        type="detection_filter_node.py"
        name="manipulator_filter"
        >
        <remap from="raw_points" to="point"/>

        <param name="filter_tolerance" type="double" value="0.10"/>
    </node>

</launch>
